### PR TITLE
Additions to the portability service documentation

### PR DESCRIPTION
**Pull request by mckinsel**

### CODE REVIEW

File: content/build/development-guides/adding-an-environment-to-the-portability-service.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+path: "/build/development-guides/adding-an-environment-to-the-portability-service.md"
+date: "2018-07-12"
+title: "Adding an Environment to the Portability Service"
+---
+
+## Why add an environment to the portability service?
+
+HCA pipelines and other user-submitted pipelines are executed in multiple environments during development and release to demonstrate their portability. The environments in which these pipelines run benefit in two ways:
```
Comment (jodihir) [inline content/build/development-guides/adding-an-environment-to-the-portability-service.md]: This page seems like a good addition to me.  Just a couple of small comments:
change this sentence to say: This testing is beneficial in two respects

File: content/build/development-guides/adding-an-environment-to-the-portability-service.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+1. Pipeline developers get quick feedback about limitations they may be introducing into their pipelines that make them incompatible with a particular environment, so they are more likely to write pipelines that execute successfully in the tested environments.
+2. Execution environment implementers get feedback about what kinds of pipelines do or do not work in their environments. With this information, they can identify potential issues with or enhancements to their environments.
+
+
+## Adding an execution environment
+
+The portability service only makes workflow execution requests using the [GA4GH WES API](https://github.com/ga4gh/workflow-execution-service-schemas). So the first step to adding an environment is to make sure that the environment implements WES.
```
Comment (jodihir) [inline content/build/development-guides/adding-an-environment-to-the-portability-service.md]: I'll add a link for WES (from that GA4GH repo README) and maybe a sentence that explains it

**Comment by jodihir** [comment]
This is ready to be pulled in.

**Comment by NoopDog** [comment]
This has been merged.

**Pull request closed without merging**